src/utils/getCopyToClipboardContents.ts

```diff
@@ -0,0 +1,29 @@
+import generateMarkDownTree from "./generateMarkDownTree";
+import { Core } from "../tree/types";
+
+/**  Will be the MarkDownTree without the deletedCore's (Any core with deletedOrder > -1)
+ * @param {Core[]} treeCore  The whole MarkDownTree
+ * @returns {string} the MarkDownTree without the deletedCore's
+ */
+
+const generateTreeCore = (treeCore: Core[]): Core[] => {
+  const visibleCoresAsLines: Core[] = [];
+
+  treeCore.forEach((core) => {
+    if (core.deletedOrder === -1) {
```

Comment:
-1 is a magic number use const NO_DELETIONS or NOT_DELETED etc to convery meaning